src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -5,32 +5,35 @@ title: New Relic Azure Monitor integration
 
-Monitor all your Azure metrics from all supported Azure services with our Azure Monitor integration.
+## Features [#features]
 
-Once you enable the New Relic Azure Monitor integration, New Relic queries your Azure platform services according to the configured polling interval. This enables all of the following features:
+Our Azure Monitor integration monitors all your Azure metrics from all supported Azure services. Once you enable it, we query your Azure platform services according to the configured polling interval. This enables all of the following features:
 
-- Metric ingestion from [all Azure services supported by Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported)
-- Native [dimensional metrics](/docs/data-apis/understand-data/new-relic-data-types/#metrics-new-relic) experience for queries, alerts and data explorer.
-- Azure [entities](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/) in the [New Relic explorer](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts/) with golden metrics and built-in dashboards.
-- Metrics decorated with custom tags defined in Azure resources
-- Faster [polling intervals](/docs/infrastructure/microsoft-azure-integrations/getting-started/polling-intervals-azure-integrations/) available (up to `1min`), enabling faster time-to-glass for metrics
-- Ability to filter monitored resources by group, type and tags
+* Metric ingestion from [all Azure services supported by Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported).
+* Native [dimensional metrics](/docs/data-apis/understand-data/new-relic-data-types/#metrics-new-relic) experience for queries, alerts, and data explorer.
+* Azure [entities](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic) in the [New Relic explorer](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts) with golden metrics and built-in dashboards.
+* Metrics decorated with custom tags defined in Azure resources.
+* Faster [polling intervals](/docs/infrastructure/microsoft-azure-integrations/getting-started/polling-intervals-azure-integrations) available (up to `1min`), enabling faster time-to-glass for metrics.
+* Ability to filter monitored resources by group, type, and tags.
 
-## Why Azure Monitor over traditional Azure integrations?
-Before the Azure Monitor integration, monitoring azure required service-specific Azure APIs to retreive metrics and metadata.
+## Why Azure Monitor over our other Azure integrations?
+
+Before the Azure Monitor integration, monitoring Azure required service-specific Azure APIs to retrieve metrics and metadata.
```

Comment:
Do we need to describe what came before? I'd rather just jump right into why I should use this :) 

Maybe the context can come after that question is answered, if you feel it's necessary